docs/static/persistent-queues.asciidoc

```diff
@@ -56,6 +56,10 @@ queue will be sized at the value of `queue.max_bytes` specified in
 `logstash.yml`. 
 The default is 1024mb (1gb).
 +
+Be sure that your disk has sufficient capacity to handle the cumulative total of `queue.max_bytes` across all persistent queues.
```

Comment:
Please consider prefixing this info with `NOTE:` or even `IMPORTANT:`.  This is good conceptual information that will expand users understanding. 

I see that this info is followed by a tip, but both of those admonitions seem appropriate and not out of place.